data/emission_factors.csv

```diff
@@ -1,100 +1,307 @@
+198,TR,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.4261,
+199,TM,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.42542,
+200,TC,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.79231,
+201,UG,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.02414,
+202,UA,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.20375,
+203,AE,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.41789,
+204,GB,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.22499,
+205,US,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.40706,
+206,VI,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.76967,
+207,UY,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.07148,
+208,UZ,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.46613,
+209,VU,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.56977,
+210,VE,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.19141,
+211,VN,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.37789,
+212,EH,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.79231,
+213,YE,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.58073,
+214,ZM,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.06617,
+215,ZW,electricity,,Carbon Footprint Ltd,Total production fuel mix factor,kWh,,,production fuel mix,kg/kWh,0.39607,
+216,global,public_transport,bus,mobitool,Stadtbus (9m),P.km,small,local,cng,kg/P.km,0.1464,
+217,global,public_transport,bus,mobitool,Stadtbus (13m),P.km,medium,local,cng,kg/P.km,0.1034,
+218,global,public_transport,bus,mobitool,Stadtbus (18m),P.km,large,local,cng,kg/P.km,0.0709,
+219,global,public_transport,bus,mobitool,Stadtbus (13m),P.km,average,local,cng,kg/P.km,0.1034,
+220,global,public_transport,bus,mobitool,Stadtbus (13m),P.km,average,local,diesel,kg/P.km,0.1338,
+221,global,public_transport,bus,mobitool,Stadtbus (13m),P.km,medium,local,diesel,kg/P.km,0.1338,
+222,global,public_transport,bus,mobitool,Stadtbus (18m),P.km,large,local,diesel,kg/P.km,0.0905,
+223,global,public_transport,bus,mobitool,Stadtbus (9m),P.km,small,local,diesel,kg/P.km,0.1716,
+224,global,public_transport,bus,mobitool,Reisebus,P.km,average,long-distance,diesel,kg/P.km,0.0465,

```

Comment:
just double-checking: Is this the only long-distance bus we have?

Reply:
i added a large one. we got large and small.